amt/charts/mps/multiTenancyMiddleware.js

```diff
@@ -25,61 +25,64 @@
                 return sendUnauthorizedResponse('ActiveProjectID header not found')
             }
 
-            // Check for auth header and decode access roles
-            const authHeader = req.get('Authorization')
-            if ( isNullOrEmpty(authHeader) || isUndefined(authHeader) ) {
-                return sendUnauthorizedResponse('Unauthorized request')
-            }
-            const authHeaderContents = authHeader.split(" ")
-            if ( authHeaderContents.length !== 2 ) {
-                return sendUnauthorizedResponse('Malformed authorization header')
-            }
-            if ( authHeaderContents[0].toLowerCase() !== 'bearer' ) {
-                return sendUnauthorizedResponse('Invalid authorization header')
-            }
-
-            const tokenContents = authHeaderContents[1].split(".")
-            const claims = Buffer.from(tokenContents[1], 'base64').toString()
-            const claimPayload = JSON.parse(claims)
-            const access = claimPayload.realm_access.roles
-            const accessList = new String(access)
-            const accessRoles = accessList.split(",")
-            if ( isNullOrEmpty(accessRoles) || isUndefined(accessRoles) ) {
-                return sendUnauthorizedResponse('Malformed token')
-            }
+            const userAgent = req.get('User-Agent')
+            if ( userAgent !== 'dm-manager') {
```

Comment:
what happens if the header is not present? Just double checking if it is safe

Reply:
If the header is not present, then `userAgent` will be empty so it will enter the if statement